### PR TITLE
Mark support assemblies as non test assemblies

### PR DESCRIPTION
**Pull request by ViktorHofer**
Fixes https://github.com/dotnet/corefx/issues/28393

### CODE REVIEW

File: src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.TestService/System.ServiceProcess.ServiceController.TestService.csproj [hunk 1]
```diff
@@ -50,7 +50,10 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <Content Include="System.ServiceProcess.ServiceController.TestService.runtimeconfig.json">
+    <Content Condition="'$(TargetNetCoreApp)' == 'true'" Include="System.ServiceProcess.ServiceController.TestService.runtimeconfig.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Condition="'$(TargetNetFx)' == 'true'" Include="System.ServiceProcess.ServiceController.TestService.exe.config">
```
Comment (ViktorHofer) [inline src/System.ServiceProcess.ServiceController/tests/System.ServiceProcess.ServiceController.TestService/System.ServiceProcess.ServiceController.TestService.csproj]: @danmosemsft @Anipik I believe you both worked in that area most of the time. You might be interested in the fact that you never tested against the netfx assemblies from corefx 😆

**Comment by ViktorHofer** [comment]
@dotnet-bot test this please

**Comment by stephentoub** [comment]
@ViktorHofer, what's the status of this PR?

**Comment by ViktorHofer** [comment]
The attempt I followed here was only a half baked solution. Ideally we just want to enable the auto referencing for such support assemblies and not bring in the whole testing infrastructure. With my upcoming arcade testing framework changes this will be resolved.

**Pull request closed without merging**